tbx/core/blocks.py

```diff
@@ -232,6 +132,19 @@ class Meta:
         template = "patterns/molecules/streamfield/blocks/video_block.html"
 
 
+class ExternalStoryEmbedBlock(WebstoryExternalStoryEmbedBlock):
+    """This code is no longer in use, unfortunately an early migration depends explicitly on it & migrations cannot be simply squashed to get around this"""
```

Comment:
**Suggestion**: it would be good to

- reference the actual migration that depends on it
- add one or two relevant links which highlight this problem (since others have encountered it too)

Reply:
Great idea! Thanks